db/migrate/20210202214540_create_flipper_tables.rb

```diff
@@ -0,0 +1,22 @@
+class CreateFlipperTables < ActiveRecord::Migration[6.1]
+  def self.up
```

Comment:
Can we use a `change` method instead?

Reply:
I think it is better this way, allowing to rollback.

Reply:
Using change your migration is still reversible, but I think you have a point 😄

Reply:
Yeah, I know what you mean... Maybe this case is reversible cause it is adding 2 tables, but I don't see harm in having the `up` and `down` methods anyway.

Reply:
Does `down` work correctly? We are also creating indexes but I guess `drop_table` will cascade those

Reply:
Yep, it's working, `drop_table` handles everything.